src/Aspire.Hosting.Garnet/GarnetBuilderExtensions.cs

```diff
@@ -0,0 +1,147 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Globalization;
+using Aspire.Hosting.ApplicationModel;
+using Aspire.Hosting.Garnet;
+using Aspire.Hosting.Utils;
+
+namespace Aspire.Hosting;
+
+/// <summary>
+/// Provides extension methods for adding Garnet resources to the application model.
+/// </summary>
+public static class GarnetBuilderExtensions
+{
+    private const string GarnetContainerDataDirectory = "/data";
+
+    /// <summary>
+    /// Adds a Garnet container to the application model.
+    /// </summary>
+    /// <example>
+    /// <remarks>Use in AspireHost</remarks>
```

Comment:
I don't think `<remarks />` is a child element of `<example />` What is `AspireHost` in this context -- or do you mean _application host_?

Reply:
OK so I've look through the XML doc comments. It's great that you've got some examples in there. I think we just need the XML doc comments applied consistently. Example:

```csharp
/// <summary>A short summary of what the type/member is.</summary>
/// <remarks>
/// <para>Some longer form content with background information if
/// it would provide value.</para>
/// <para>You can use the para element to break the text up. This
/// text would not necessarily have code but would use plenty
/// of references.</para>
/// <example>
/// A description of the example and what it shows.
/// <code>
/// The actual code.
/// </code>
/// </example>
/// <example>
/// Another example with a description.
/// <code>
/// The actual code.
/// </code>
/// </example>
```

Reply:
Looks like I fixed everything